Examples/Modules/nuclear_fusion/analysis_deuterium_tritium_fusion.py

```diff
@@ -330,24 +344,25 @@ def check_fusion_yield(data, expected_fusion_number, E_com, reactant0_density, r
         assert(np.all(is_close(product_weight_theory, product_weight_simulation,
                                rtol = 5.*relative_std_weight)))
 
-def specific_check1(data):
-    check_isotropy(data, relative_tolerance = 3.e-2)
+def specific_check1(data, dt):
+    if not is_RZ:
+        check_isotropy(data, relative_tolerance = 3.e-2)
```

Comment:
We don't check for isotropy in the RZ case, since the distribution of momentum for the products is not yet correct.